### PR TITLE
Remove Connect Wallet Logic from home screen

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#364** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/364?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/home/TokenBalanceList.tsx [hunk 1]
```diff
@@ -49,7 +49,8 @@ const TokenBalanceItem = ({
 } & Omit<LinkProps, 'children'>) => {
   // const { data: sendAccounts } = useSendAccounts()
   // const sendAccount = sendAccounts?.[0]
-  const sendAccount = useAccount()
+  const { data: addresses } = useChainAddresses()
```
Comment (0xBigBoss) [inline packages/app/features/home/TokenBalanceList.tsx:52]: did we add this create this branch from dev? weird it's tracking this as a change. This should already be in main.

File: packages/app/features/home/TokenDetails.tsx
Comment (0xBigBoss) [inline packages/app/features/home/TokenDetails.tsx:1]: ty

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/home/TokenBalanceList.tsx (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/features/home/TokenBalanceList.tsx:52]: Not sure what you mean

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Apr 28, 7:59 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/364).

**Pull request merged**